pipelines/tests/test_pipelines.py

```diff
@@ -0,0 +1,29 @@
+import unittest
+import os
+import sys
+
+from dwave.system.samplers import DWaveSampler
+
+try:
+    DWaveSampler()
+    _config_found = True
+except ValueError:
+    _config_found = False
+
+# /tests/integration/test_demo.py
+PY2 = sys.version_info.major == 2
```

Comment:
Is there a particular part of the code that relies on Python 2?

The demos should support Python 2.7 and 3.5+. Hence, CircleCI will run this smoke test in all of those python environments.